WhaleSpotting.UnitTests/Controllers/SightingControllerTests.cs

```diff
@@ -38,5 +42,83 @@ public async Task GetInfo_Called_ReturnsSightings()
             // Assert
             result.Should().HaveCount(2);
         }
+
+        [Fact]
+        public void CreateSighting_CalledWithNewSighting_ReturnsCreatedResult()
+        {
+            // Arrange
+            var newSighting = new SightingRequestModel
+            {
+                Species = Species.AtlanticWhiteSidedDolphin,
+                Quantity = 2,
+                Description = "was nice",
+                Longitude = -100.010,
+                Latitude = -22.010,
+                Location = "atlantic ocean",
+                SightedAt = DateTime.Now,
+                OrcaType = null,
+                OrcaPod = "",
+                UserId = 5,
+            };
+
+            var sightingResponse = new SightingResponseModel
+            {
+                Id = 1,
+                SightedAt = DateTime.Now,
+                Species = "AtlanticWhiteSidedDolphin",
+                Quantity = 2,
+                Location = "atlantic ocean",
+                Longitude = -100.010,
+                Latitude = -22.010,
+                Description = "was nice",
+                OrcaType = "",
+                OrcaPod = "",
+                UserId = 5,
+                Username = "FakeUser",
+                Confirmed = false,
+            };
+
+            A.CallTo(() => _sightings.CreateSighting(newSighting))
+                .Returns(sightingResponse);
+
+            //Act
```

Comment:
Can we have spaces before the As (e.g. `// Act`)

Reply:
Addressed.